src/Controls/src/Core/ClickGestureRecognizer.cs

```diff
@@ -1,80 +0,0 @@
-using System;
-using System.ComponentModel;
-using System.Windows.Input;
-
-namespace Microsoft.Maui.Controls
-{
-	/// <include file="../../docs/Microsoft.Maui.Controls/ButtonsMask.xml" path="Type[@FullName='Microsoft.Maui.Controls.ButtonsMask']/Docs" />
-	[Flags]
-	public enum ButtonsMask
-	{
-		/// <include file="../../docs/Microsoft.Maui.Controls/ButtonsMask.xml" path="//Member[@MemberName='Primary']/Docs" />
-		Primary = 1 << 0,
-		/// <include file="../../docs/Microsoft.Maui.Controls/ButtonsMask.xml" path="//Member[@MemberName='Secondary']/Docs" />
-		Secondary = 1 << 1
-	}
-
-	/// <include file="../../docs/Microsoft.Maui.Controls/ClickGestureRecognizer.xml" path="Type[@FullName='Microsoft.Maui.Controls.ClickGestureRecognizer']/Docs" />
-	public sealed class ClickGestureRecognizer : GestureRecognizer
```

Comment:
This was only wired up to `GTK` and `macOS` in XF. Currently in .NET MAUI this gesture does nothing anywhere. I think just condensing down into the `TapGestureRecognizer` here makes the most sense.